indexing-solr/src/test/resources/datastream/techmd_no_duration.xml

```diff
@@ -0,0 +1,105 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<premis3:premis xmlns:premis3="http://www.loc.gov/premis/v3">
+    <script/>
+    <premis3:object xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:type="premis3:file">
+        <premis3:objectCharacteristics>
+            <premis3:compositionLevel>0</premis3:compositionLevel>
+            <premis3:format>
+                <premis3:formatDesignation>
+                    <premis3:formatName>MPEG-PS</premis3:formatName>
+                </premis3:formatDesignation>
+            </premis3:format>
+            <premis3:size>116736</premis3:size>
+            <premis3:objectCharacteristicsExtension>
+                <fits xmlns="http://hul.harvard.edu/ois/xml/ns/fits/fits_output" xsi:schemaLocation="http://hul.harvard.edu/ois/xml/ns/fits/fits_output http://hul.harvard.edu/ois/xml/xsd/fits/fits_output.xsd" version="1.6.0" timestamp="7/9/24, 4:38 PM">
+                    <identification status="CONFLICT">
+                        <identity format="MPEG-PS" mimetype="video/mpg" toolname="FITS" toolversion="1.6.0">
+                            <tool toolname="MediaInfo" toolversion="23.09"/>
+                        </identity>
+                        <identity format="symbolic link to `/opt/data/deposits/31d00601-78cd-480f-b9e8-9b319790f7a4/data/_a49114ad-2276-4e99-afda-67c2fa8f0a24'" mimetype="inode/symlink" toolname="FITS" toolversion="1.6.0">
+                            <tool toolname="file utility" toolversion="5.11"/>
+                        </identity>
+                    </identification>
+                    <fileinfo>
+                        <filepath toolname="OIS File Information" toolversion="1.0" status="SINGLE_RESULT">/path/to/video.mpeg</filepath>
+                        <filename toolname="OIS File Information" toolversion="1.0" status="SINGLE_RESULT">video.mpeg</filename>
+                        <size toolname="MediaInfo" toolversion="23.09">116736</size>
+                        <md5checksum toolname="OIS File Information" toolversion="1.0" status="SINGLE_RESULT">5f7973ff9e4152827994d4149c8af39d</md5checksum>
+                        <fslastmodified toolname="OIS File Information" toolversion="1.0" status="SINGLE_RESULT">1720543091535</fslastmodified>
+                    </fileinfo>
+                    <filestatus/>
+                    <metadata>
+                        <video>
+                            <location toolname="MediaInfo" toolversion="23.09" status="SINGLE_RESULT">/opt/data/deposits/31d00601-78cd-480f-b9e8-9b319790f7a4/data/_a49114ad-2276-4e99-afda-67c2fa8f0a24</location>
+                            <mimeType toolname="MediaInfo" toolversion="23.09" status="SINGLE_RESULT">video/mpg</mimeType>
+                            <format toolname="MediaInfo" toolversion="23.09" status="SINGLE_RESULT">MPEG-PS</format>
+                            <duration toolname="MediaInfo" toolversion="23.09" status="SINGLE_RESULT">467</duration>
```

Comment:
If we start using the video/duration element then we will probably need to remove this element for the no duration test.